1-js/03-code-quality/06-polyfills/article.md

```diff
@@ -3,37 +3,37 @@
 
 ## Babel
 
-Quando usamos funcionalidades modernas da linguagem, algumas engines podem não suportar esse código. Como dito anteriormente, nem todas as funcionalidades são implementadas em todos os lugares.
+Quando usamos funcionalidades modernas da linguagem, alguns interpretadores _(engines)_ podem não suportar esse código. Como dito anteriormente, nem todas as funcionalidades são implementadas em todos os lugares.
 
 É aqui que o Babel vem para nos ajudar.
 
-[Babel](https://babeljs.io) é um [transpiler](https://en.wikipedia.org/wiki/Source-to-source_compiler). Ele reescreve um código JavaScript moderno em um padrão antigo.
+[Babel](https://babeljs.io) é um [transpiler](https://en.wikipedia.org/wiki/Source-to-source_compiler). Ele reescreve um código em JavaScript moderno em um padrão antigo.
 
 Atualmente, o Babel possui duas partes:
 
 1. Primeiro, o programa responsável pelo _transpile_, onde o código é reescrito. O desenvolvedor roda o programa na própria máquina, o Babel reescreve o código em um padrão antigo e, após isso, o codigo é entregue ao website para os usuários. Sistemas de build modernos como [webpack](http://webpack.github.io/) ou [brunch](http://brunch.io/) fornecem meios para executar o transpiler automaticamente em todas as mudanças realizadas no código, de forma que não perdemos tempo do nosso lado.
 
 2. Segundo, o polyfill.
 
-    O transpiler reescreve o código, então funcionalidades de sintaxe são cobertas. Porém, para novas funcionalidades, precisamos escrever um script especial que cuida dessa implementação. O JavaScript é uma linguagem altamente dinâmica, onde scripts podem não apenas adicionar novas funcionalidades, mas também modificar as já existentes, de forma que elas se comportem de acordo com o padrão moderno.
+    O transpiler reescreve o código, então funcionalidades de sintaxe são cobertas. Porém, para novas funcionalidades, precisamos de escrever um script especial que cuida dessa implementação. O JavaScript é uma linguagem altamente dinâmica, onde scripts podem não apenas adicionar novas funcionalidades, mas também modificar as já incorporadas _(built-in)_ na linguagem, de forma a que elas se comportem de acordo com o padrão moderno.
 
     Existe o termo "polyfill" para scripts que "preenchem" essa lacuna e adicionam as implementações que faltam.
 
     Dois polyfills interessantes são:
     - [babel polyfill](https://babeljs.io/docs/usage/polyfill/) que suporta várias funcionalidades, mas é bem grande.
     - [polyfill.io](http://polyfill.io) serviço que permite carregar/construir polyfills sob demanda, dependendo das funcionalidades que precisamos.
 
-Então, precisamos configurar o transpiler e adicionar o polyfill para fazer com que as engines antigas suportem novas funcionalidades.
+Então, precisamos configurar o transpiler e adicionar o polyfill para fazer com que os interpretadores _(engines)_ antigos suportem novas funcionalidades.
```

Comment:
Please,
 - 'Então, precisamos de configurar o transpiler ...'
- do not italicize the parentheses.